pkg/cmd/addon/create/cmd.go

```diff
@@ -0,0 +1,54 @@
+// Copyright Contributors to the Open Cluster Management project
+package create
+
+import (
+	"fmt"
+
+	genericclioptionsclusteradm "open-cluster-management.io/clusteradm/pkg/genericclioptions"
+	clusteradmhelpers "open-cluster-management.io/clusteradm/pkg/helpers"
+
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+var example = `
+Create an addon from manifests by using AddonTemplate
+%[1]s addon create helloworld -f deployment.yaml
+`
+
+// NewCmd creates a cammand to create an addon
+func NewCmd(clusteradmFlags *genericclioptionsclusteradm.ClusteradmFlags, streams genericclioptions.IOStreams) *cobra.Command {
+	o := NewOptions(clusteradmFlags, streams)
+
+	cmd := &cobra.Command{
+		Use:          "create",
+		Short:        "create a specified addon",
+		Long:         "create a specific add-on(s) on the hub",
+		Example:      fmt.Sprintf(example, clusteradmhelpers.GetExampleHeader()),
+		SilenceUsage: true,
+		PreRunE: func(c *cobra.Command, args []string) error {
+			clusteradmhelpers.DryRunMessage(clusteradmFlags.DryRun)
+
+			return nil
+		},
+		RunE: func(c *cobra.Command, args []string) error {
+			if err := o.complete(c, args); err != nil {
+				return err
+			}
+			if err := o.Validate(); err != nil {
+				return err
+			}
+			if err := o.Run(); err != nil {
+				return err
+			}
+
+			return nil
+		},
+	}
+
+	cmd.Flags().StringVarP(&o.Version, "name", "v", "0.0.1", "Specified version of the addon")
```

Comment:
```
clusteradm addon create --help

panic: unable to redefine 'v' shorthand in "create" flagset: it's already used for "name" flag
```

The `-v` here conflicts with the global option `-v`(number for the log level verbosity)

Reply:
```suggestion
	cmd.Flags().StringVar(&o.Version, "version", "0.0.1", "Specified version of the addon")
```